docs/app/index.ejs

```diff
@@ -124,6 +124,8 @@
       font-size: 87.5%;
       background-color: rgba(0, 0, 0, 0.04);
       border-radius: 3px;
+      white-space: pre;
+      display: inline-block;
```

Comment:
With proper whitespace, now the `padding-top` and `padding-bottom` need to be adjusted to match.  The block element now has too much bottom padding.

Let's go with `padding: 0.1em 0` instead of the 3 rules currently in place (see line 120-123).

Reply:
No problem. After looking at it some more though, it looks like with `padding: 0.1em 0`, `display: inline-block` pushes the text up. Try setting both those rules in e.g. the Icon docs, then toggle the `display` setting on and off in dev tools - you'll see the text bounce up and down. If we use `padding: 0` though, the `code` padding will still look larger but without pushing the text around.

It's your call though - just let me know.

Reply:
I went ahead and added the `padding: 0.1em 0` as a separate commit. Let me know if you want me to change it or squash them.

Reply:
I see what you're saying.  I think `padding: 0;` looks great.  In fact, removing all padding rules may be the same thing in this case.  Try it with no padding rules and see what you think.

I'll let you make the call and we'll ship it.

Reply:
A good point. Let's go with that